source/extensions/common/wasm/context.cc

```diff
@@ -56,6 +55,12 @@ namespace Wasm {
 
 namespace {
 
+// Register the custom namespace which prefixes all the user-defined metrics.
+// Note that the prefix is removed from the final output of /stats endpoints.
+constexpr absl::string_view CustomMetricNamespace = "wasmcustom";
+constexpr absl::string_view CustomMetricNamespacePrefix = "wasmcustom.";
+Stats::Utility::RegisterCustomStatNamespace RegisterNamespace{CustomMetricNamespace};
```

Comment:
Can we do this when initializing the extension, rather than as a style-guide violating static non-pod?